java-frontend/src/test/java/org/sonar/java/model/declaration/ClassTreeImplTest.java

```diff
@@ -87,6 +87,7 @@ public void visitVariable(VariableTree tree) {
   void records_members_order() {
     ClassTree classTree = (ClassTree) JParserTestUtils.parse(
       "record Output(String title, String summary, String text) {\n"
+        + "  public Output {}"
```

Comment:
Nice addition, it is a shame we did not test compact constructors before

Reply:
:+1: